common/scripted_effects/00_political_setup.txt

```diff
@@ -55,6 +57,7 @@ effect_starting_politics_reactionary = {
 	activate_law = law_type:law_professional_army
 	activate_law = law_type:law_expansionism
 	activate_law = law_type:law_tenant_farmers
+	activate_law = law_type:law_unitary
 }
 
 effect_starting_politics_traditional = {
```

Comment:
missing starting law

Reply:
the default law in the centralization group is informal authority so it should be fine not to include it in effect_starting_politics_traditional but i can add it just so it's consistent

Reply:
The convention with the other laws is to add them anyway, so either remove all the redundant laws, or just add yours to the list.

Reply:
Done